test/fixtures/hello-cloud/template-runner.yml

```diff
@@ -12,6 +12,7 @@ template:
       name: runner
       app: hello-cloud
   spec:
+    restartPolicy: Never
```

Comment:
I had to add this to get `test_run_with_verify_result` to pass. Otherwise the deploy failed with: 
```
INFO][2018-07-20 08:55:11 -0700]	---------------------------------Phase 1: Validating configuration----------------------------------
[INFO][2018-07-20 08:55:11 -0700]	
[INFO][2018-07-20 08:55:11 -0700]	----------------------------------Phase 2: Fetching task template-----------------------------------
[INFO][2018-07-20 08:55:11 -0700]	Fetching task runner pod template: 'hello-cloud-template-runner' in namespace: 'k8sdeploy-test-run-with-verify-result-ec88a252fd86931b'
[INFO][2018-07-20 08:55:11 -0700]	
[INFO][2018-07-20 08:55:11 -0700]	---------------------------Phase 3: Constructing final pod specification----------------------------
[INFO][2018-07-20 08:55:11 -0700]	Rendering template for task runner pod
[INFO][2018-07-20 08:55:11 -0700]	
[INFO][2018-07-20 08:55:11 -0700]	---------------------------------------Phase 4: Creating pod----------------------------------------
[INFO][2018-07-20 08:55:11 -0700]	Starting task runner pod: 'task-runner-b9df5e758ecfb97d'
[ERROR][2018-07-20 08:55:21 -0700]	Pod/task-runner-b9df5e758ecfb97d failed to deploy after 9.2s
[INFO][2018-07-20 08:55:21 -0700]	
[INFO][2018-07-20 08:55:21 -0700]	------------------------------------------Result: FAILURE-------------------------------------------
[FATAL][2018-07-20 08:55:21 -0700]	Failed to deploy 1 resource
[FATAL][2018-07-20 08:55:21 -0700]	
[FATAL][2018-07-20 08:55:21 -0700]	Pod/task-runner-b9df5e758ecfb97d: FAILED
[FATAL][2018-07-20 08:55:21 -0700]	The following containers encountered errors:
[FATAL][2018-07-20 08:55:21 -0700]	> task-runner: Crashing repeatedly (exit 0). See logs for more information.
[FATAL][2018-07-20 08:55:21 -0700]	
[FATAL][2018-07-20 08:55:21 -0700]	  - Final status: Running
[FATAL][2018-07-20 08:55:21 -0700]	  - Events (common success events excluded):
[FATAL][2018-07-20 08:55:21 -0700]	      [Pod/task-runner-b9df5e758ecfb97d]	SuccessfulMountVolume: MountVolume.SetUp succeeded for volume "default-token-dg6j9"  (1 events)
[FATAL][2018-07-20 08:55:21 -0700]	      [Pod/task-runner-b9df5e758ecfb97d]	BackOff: Back-off restarting failed container (2 events)
[FATAL][2018-07-20 08:55:21 -0700]	  - Logs from container 'task-runner' (last 250 lines shown):
[FATAL][2018-07-20 08:55:21 -0700]	      KUBERNETES-DEPLOY
```